vector/src/main/java/im/vector/app/features/home/room/detail/RoomDetailViewModel.kt

```diff
@@ -1370,6 +1396,7 @@ class RoomDetailViewModel @AssistedInject constructor(
         if (vectorPreferences.sendTypingNotifs()) {
             room.userStopsTyping()
         }
+        chatEffectManager.dispose()
```

Comment:
Also set the delegate to `null`? I see that it is done in `dispose()`, but I prefer to see this call here, for symmetry.

Reply:
Done